File: rb/test/exists_test.rb
# LmMultichannel SDK exists test

require "minitest/autorun"
require_relative "../LmMultichannel_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = LmMultichannelSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { LmMultichannelSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await LmMultichannelSDK.test()
    equal(null !== testsdk, true)
  })

})
